v2/dns.go

```diff
@@ -45,9 +45,9 @@ func (c *DNS) DNSSearchPaths(ctx context.Context) ([]string, error) {
 	return resp["searchPaths"], nil
 }
 
-// SetDNSNameservers replaces the list of DNS nameservers for the given tailnet with the list supplied by the user. Note
+// SetNameservers replaces the list of DNS nameservers for the given tailnet with the list supplied by the user. Note
 // that changing the list of DNS nameservers may also affect the status of MagicDNS (if MagicDNS is on).
-func (c *DNS) SetDNSNameservers(ctx context.Context, dns []string) error {
+func (c *DNS) SetNameservers(ctx context.Context, dns []string) error {
```

Comment:
Not sure if you already did a scan, but would be nice to have these names match our api docs

Reply:
BTW, I assume you're referring to the `operationId`? If so, we have tow problems:

1. The operationId is `setDnsNameservers`, which stutters
2. The operationId doesn't appear to actually be shown to the user anywhere in the openapi docs (not even in URLs)